docs/install.rst

```diff
@@ -50,15 +116,15 @@ Here's a sample output from the command:
 
    $ pg_autoctl show systemd
```

Comment:
`pg_autoctl show systemd` errors out when `--pgdata` is not provided. We are not setting `--pgdata` in the tutorial. Perhaps we should either set environment variable or provide the value

Reply:
Fixed!